presto-main/src/main/java/io/prestosql/connector/system/QuerySystemTable.java

```diff
@@ -55,6 +56,8 @@
     public static final ConnectorTableMetadata QUERY_TABLE = tableMetadataBuilder(QUERY_TABLE_NAME)
             .column("query_id", createUnboundedVarcharType())
             .column("state", createUnboundedVarcharType())
+            .column("error_type", createUnboundedVarcharType())
```

Comment:
I wonder if new columns shouldn't be added as last to not break existing system runtime queries. On the other hand, if someone depends on column index, that's probably a bad practice

Reply:
I think I can move it anyway. It does matter to me.